Sources/ObjCSupport/LOKLabelLayout.swift

```diff
@@ -9,6 +9,15 @@
 import UIKit
 
 @objc open class LOKLabelLayout: LOKBaseLayout {
+    @objc public let attributedString: NSAttributedString?
+    @objc public let string: String?
+    @objc public let font: UIFont
+    @objc public let numberOfLines: Int
+    @objc public let alignment: LOKAlignment
+    @objc public let viewClass: UILabel.Type
+    @objc public let configure: ((UILabel) -> Void)?
+    public let lineHeight: CGFloat?
```

Comment:
This property, along with `maxWidth` and `maxHeight`, should probably be `@objc` and non-optional.